packages/scene-graph-cocos/client/src/exporter/scene/DefaultSceneExporter.ts

```diff
@@ -450,9 +448,13 @@ export default class CocosCreator implements SceneExporter {
       return null;
     }
 
-    for (const [key, value] of resourceMap.entries()) {
-      const entity = value;
+    const it = resourceMap.entries();
```

Comment:
Fixed TypeScript transpile error.
Iterator object can not be iterated with for...of.